1.train_models/log_reg_plate_2_cp_fs_data/nbconverted/train_model.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
```

Comment:
I see a lot of redundant code here (redundant with the files in plate 1).

Remember [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is best practices! Is there a way to refactor out functions in this code to help? (Sometimes DRY can be violated, in some circumstances)

Reply:
Ya, I was thinking about that. I was trying to make each model and analysis as modular as possible.

In some cases the data wrangling may be slightly different for each dataset, but now that you mention it, I think it's a good idea to break this structure, in this case, and refactor with some functions for plates 1 and 2.